sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosContainerTest.java

```diff
@@ -264,6 +258,40 @@ public void readAllContainers() throws Exception{
         assertThat(feedResponseIterator1.iterator().hasNext()).isTrue();
     }
 
+    @Test(groups = { "emulator" }, timeOut = TIMEOUT)
```

Comment:
Please cover query, feed tests as well.

Reply:
does the current public emulator supports this feature?

Reply:
Yes, public emulator supports subparitioning.